app/src/main/java/ru/practicum/android/diploma/favorite/domain/FavoriteTrack.kt

```diff
@@ -0,0 +1,7 @@
+package ru.practicum.android.diploma.favorite.domain
+
+data class FavoriteTrack(
+
+    private val url:String
+
+)
```

Comment:
```suggestion
data class FavoriteTrack(private val url:String)
```

Это все лучше писать в 1 строку